.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```

Comment:
EOF = End of File, you should add one blank line to the end of the file. That's why GitHub shows this icon:
![image](https://user-images.githubusercontent.com/36927617/126648759-5e0d9df6-77d1-4b2f-9518-ecdd553a96ce.png)